### PR TITLE
Updated maintainers, Bumped version.

### PR DESCRIPTION
**Pull request by yuce**

### CODE REVIEW

File: src/pot.app.src [hunk 1]
```diff
@@ -26,7 +26,10 @@
   {env,[]},
   {modules, []},
 
-  {maintainers, ["Yuce Tekol"]},
+  {maintainers, [
```
Comment (nalundgaard) [inline src/pot.app.src]: I think we should remove this, the field is deprecated in Hex and results in a publish error on the latest versions of rebar3: https://github.com/tsloughter/rebar3_hex/issues/125

File: README.md [hunk 2]
```diff
@@ -461,6 +466,8 @@ NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FO
 DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
+## Useful Links
```
Comment (nalundgaard) [inline README.md]: Does this render? This looks like a markdown links list, I don’t think they are displayed in the rendered output.

**Review by nalundgaard — CHANGES_REQUESTED** [review]
Couple of concerns from me called out in the line items.

**Comment by yuce** [comment]
Heck, it seems my rebar3 and hex was quite a bit behind. Added you as an owner of the package, hopefully it will work this time.

**Comment by nalundgaard** [comment]
Cool! All set, I published 1.0.0 just now.

Thanks!

**Comment by yuce** [comment]
That's great! 🎉🎉🎉

**Pull request merged**